docs/package.json

```diff
@@ -20,7 +20,7 @@
     "@tanstack/react-query": "5.20.5",
     "@vercel/analytics": "1.2.2",
     "autoprefixer": "10.4.18",
-    "cmdk": "0.2.1",
+    "cmdk": "1.0.0",
```

Comment:
With this update the colors have changed. It looks too bright for me and is now difficult to read and also difficult to recognize what the selected element is.
![Bildschirmfoto 2024-03-11 um 08 30 30](https://github.com/marigold-ui/marigold/assets/77496890/f602a7a2-a3dd-4630-86a0-2faf9659e98b)